WebApp/autoreduce_webapp/static/javascript/help.js

```diff
@@ -34,21 +34,39 @@
         }
     };
 
+    var headingTextToDashed = function headingTextToDashed(headingText) {
```

Comment:
Though the names are pretty self explanatory, i think we should add comments to explicitly describe what's happening for each var/function